src/LuaXML_101012/Makefile

```diff
@@ -1,9 +1,13 @@
 # 2009-03-16 by gf
 #
 ifeq ($(PREFIX),)
-    PREFIX := /usr
+    override PREFIX := /usr
 endif
 
+# Path for installation Lua modules
+# Path is relative to $(DESTDIR)
+override LUAPATH ?= $(PREFIX)/lib/lua
```

Comment:
Just out of pure curiosity (or lack of knowledge),
Is it not pointless to use conditional variable assignment (?=) along with override directive?
Do you think that somewhere in this file LUAPATH might be defined?

Reply:
You are right that use both `override` and `?=` is not needed and i kept only `?=`
But `override` is still needed for statements like `override CFAGS += ...` because without `override` it does not work in case of next form of `make` run
```
make CFLAGS=...
```

Reply:
fixed